src/main/webapp/deletefile.html

```diff
@@ -24,13 +24,22 @@ <h2>Delete Configuration</h2>
       <a href="editfile.html">Edit Configuration</a>
       <a href="createfile.html">Create Configuration</a>
     </div><br>
-    <!-- Should be a dropdown later on. -->
     <div id="delete-search">
-      <form onsubmit="openModal()" id="delete-form">
```

Comment:
do you really want to get rid of this modal? Even with a dropdown it's probably nice to have a confirmation

Reply:
The modal is still functioning. Just this `onsubmit="openModal()` seems to be unnecessary without a matching javascript method.